### PR TITLE
Keep extract_genomic_dna in Fetch Sequences

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

File: europe-custom.yaml [hunk 1]
```diff
@@ -242,7 +242,7 @@ tools:
     tool_panel_section_label: Extract Features
   - name: extract_genomic_dna
     owner: iuc
-    tool_panel_section_label: "Other Tools"
+    tool_panel_section_label: "Fetch Sequences / Alignments"
```
Comment (bgruening) [inline europe-custom.yaml:245]: With special characters it might be better to specify not the `tool_panel_section_label` but the `tool_panel_section_id`

Comment (bgruening) [reply europe-custom.yaml:245]: Reason is that Galaxy has some magic to convert those labels to IDs, but with special characters, you never know ...

Comment (wm75) [reply europe-custom.yaml:245]: I see. With `tool_panel_section_id` as the yaml key, then?

Comment (bgruening) [reply europe-custom.yaml:245]: Yes: https://github.com/usegalaxy-eu/usegalaxy-eu-tools/blob/master/single-cell-ebi-gxa.yaml.lock#L12

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Danke.

**Pull request merged**